src/Home/View/page.twig

```diff
@@ -6,34 +6,26 @@
 	<a href="#slide-4"></a>
 	<a href="#slide-5"></a>
 </nav>
-<section id="slide-1" class="container home-page clearfix">
-	<div class="row">
-		<div class="col-md-6"></div>
-		<div class="img-box">
-			<img class="slider-img" src="/img/slider1.svg">
-		</div>
-		<div class="col-md-5">
-			<div class="text-box">
-				<div class="text-box-img">
-					<img class="globe-img--small" src="/img/globe.svg">
-				</div>
+<section id="slide-1" class="home-page clearfix">
+	<div class="row clearfix">
+		<div class="col-md-6 img-box">
```

Comment:
Format the code. The close and open tag should have the same indent. Sometimes you use one tab and sometimes two tab (example lines: 15). Use the same indent.